### PR TITLE
dmd 2.076.0

### PR DESCRIPTION
**Pull request by John-Colvin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dmd.rb [hunk 1]
```diff
@@ -3,22 +3,22 @@ class Dmd < Formula
   homepage "https://dlang.org/"
 
   stable do
```
Comment (fxcoudert) [inline Formula/dmd.rb]: Remove the `stable do` block, which is unnecessary now.

File: Formula/dmd.rb [hunk 2]
```diff
@@ -77,63 +53,31 @@ def install
...
+    EOS
+    etc.install "dmd.conf"
   end
 
   # Previous versions of this formula may have left in place an incorrect
   # dmd.conf.  If it differs from the newly generated one, move it out of place
   # and warn the user.
-  # This must be idempotent because it may run from both install() and
-  # post_install() if the user is running `brew install --build-from-source`.
   def install_new_dmd_conf
```
Comment (fxcoudert) [inline Formula/dmd.rb]: Move the content of `install_new_dmd_conf` directly into `post_install`, since now it isn't called from any other place anymore.

**Review by fxcoudert — DISMISSED** [review]

**Review by ilovezfs — APPROVED** [review]

File: Formula/dmd.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/dmd.rb]: Note there's still a `head do` so it's actually still needed.

**Comment by ilovezfs** [comment]
Thanks for the upgrade @John-Colvin! Shipped :)

**Pull request closed without merging**